lib/puppet/provider/service/smf.rb

```diff
@@ -34,11 +34,12 @@ def setupservice
   end
 
   def self.instances
-   svcs("-H").split("\n").select{|l| l !~ /^legacy_run/ }.collect do |line|
-     state,stime,fmri = line.split(/\s+/)
```

Comment:
Moving to an explicit list of fields allows us to drop the unused stime variable